post_mod_update.sh

```diff
@@ -88,6 +93,13 @@ reply_response=$(curl -X POST -L \
 -F "topic_comment_${forum_topic_id}=" \
 -F "topic_comment_${forum_topic_id}_noscript=<messagestripped.html" \
 -A "$user_agent" -b ./cookies -c ./cookies "$forum_topic_url")
+echo "$reply_response" > ./reply_response.html
+
+error_message=$(echo "$reply_response" | pup 'div.ipsType_warning[data-role="commentFormError"] text{}')
+if ! [[ -z "$error_message" ]]; then
```

Comment:
use `-n`, like `if [[ -n "$error_message" ]]`

Reply:
Btw, on GitHub it's possible to press this button: ![image](https://github.com/Kerbalight/ksp2-forum-post-action/assets/27270190/fb3385c1-0da5-402b-a688-62d7aad3580b)

To create a small suggestion like this:
```suggestion
if [[ -n "$error_message" ]]; then
```